### PR TITLE
Fix BEM format and issue in the developer tab of input counter

### PR DESCRIPTION
**Pull request by cristain7**

### CODE REVIEW

File: src/components/form/input/input.scss [hunk 1]
```diff
@@ -48,19 +48,20 @@
...
-      padding: 0;
-      text-align: center;
-      user-select: none;
-      width: 1.25rem;
-    }
+  &__counter-button {
+    border: none;
+    background: none;
+    cursor: pointer;
+    @extend %ods-text--size-juliett;
```
Comment (larsboldt) [inline src/components/form/input/input.scss:57]: Did this pass the linter? All @extends should be at the top

File: src/components/form/input/input.js [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
-  btnUp.addEventListener('click', increaseCount, false);
-  btnDown.addEventListener('click', decreaseCount, false);
+function triggerChangeNumber(callback) {
+  const counterTriggers = document.querySelectorAll('.ods-input__counter-nav');
+  counterTriggers.forEach((item) => {
+    callback(item);
+  });
 }
 
+const OdsInput = {
```
Comment (larsboldt) [inline src/components/form/input/input.js:23]: Missing the single element bindings making this hard to use in a vue context

**Review by larsboldt — CHANGES_REQUESTED** [review]

File: src/components/form/input/input.scss (hunk 1 above)
Comment (cristain7) [reply src/components/form/input/input.scss:57]: it did, because it is in the test environment

Comment (cristain7) [reply src/components/form/input/input.scss:54]: Fixed all the  linting

File: src/components/form/input/input.js (hunk 2 above)
Comment (cristain7) [reply src/components/form/input/input.js:23]: Added it

**Review by kbergha — APPROVED** [review]

File: src/components/form/input/input.js [hunk 3]
```diff
@@ -1,41 +1,55 @@
...
+  bindElement(element) {
+    element.addEventListener('click', changeCounter, false);
+  },
+
+  unbindElement(element) {
+    element.removeEventListener('click', changeCounter, false);
+  },
+
+  bindAll() {
+    triggerChangeNumber((item) => {
```
Comment (larsboldt) [inline src/components/form/input/input.js:43]: endres til triggerIterator for å være konsistent med de andre komponentene

File: src/components/form/input/input.js [hunk 4]
```diff
@@ -1,41 +1,55 @@
...
+      item.addEventListener('click', changeCounter, false);
+    });
+  },
+
+  unbindAll() {
+    triggerChangeNumber((item) => {
```
Comment (larsboldt) [inline src/components/form/input/input.js:49]: endres til triggerIterator for å være konsistent med de andre komponentene

**Review by larsboldt — CHANGES_REQUESTED** [review]

**Review by larsboldt — APPROVED** [review]

**Pull request merged**